project/plugins.sbt

```diff
@@ -20,9 +20,9 @@ addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0" excludeAll Ex
 
 addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.3")
 
-addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.3")
+//addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.3")
 
-addSbtPlugin("net.vonbuchholtz" % "sbt-dependency-check" % "0.2.0")
+//addSbtPlugin("net.vonbuchholtz" % "sbt-dependency-check" % "0.2.0")
```

Comment:
these didn't seem to be used so were just slowing things down.  They may have just been added as a convenience when someone was doing updates/dependencies?